### DISCUSSION TITLE
Edit taxonomy view name?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#6029] [General]
Hi

My taxonomy views are showing up in the `template` fieldtype, I can hide them by adding an underscore at the start of the view filename, shown below:

```
resources/views/cms_docs/docs/_index.antlers.html
resources/views/cms_docs/docs/_show.antlers.html
```

But I've already used these terms/views. links to these pages are broken, and I can't seem to find anywhere to update them?

I've tried clearing the cache `php artisan optimize:clear`

Can someone point me in the right direction?

**Comment by edalzell** [comment]
Adding the underscore has made them a partial, not a template, that's why they don't show and they break.

What's the actual issue you're having?

**Reply by stephenmeehanuk** [reply]
I'd like to hide them from the templates field.
![Screenshot 2022-05-11 at 17 37 20](https://user-images.githubusercontent.com/72254741/167902212-16abecd3-cae3-4dfb-9691-52e62f8f0fd0.png)

**Reply by stephenmeehanuk** [reply]
As a workaround I've created a `select_field` and added the template names as keys. I've named the `select_field` `template`. It seems to work. I do wonder why the actual template fieldtype shows taxonomy templates, as these are unlikely to be used for anything else. It also shows the layout view?

**Reply by edalzell** [reply]
ya I agree there should be a way to configure what templates are shown. You can always make your own fieldtype, that's what I've done in the past when running into this.

**Reply by stephenmeehanuk** [reply]
> You can always make your own fieldtype

Unfortunately that's a bit out of my reach at the moment. I can get by with the workaround for now. I'll put in a feature request for the template field though.

**Comment by jasonvarga** [comment]
The `template` fieldtype has always felt a little silly to me. As soon as you starting wanting to say "dont show this" and "dont show that" it's really just easier to make a regular `select` fieldtype and providing the items you _do_ want to allow to be selected.


```yaml
handle: template
field:
  type: select
  options:
    pages_three_column: 3 Columns
    pages_two_column: 2 Columns
```

**Reply by stephenmeehanuk** [reply]
Thanks, that's what I ended up doing.
